includes/Core/Permissions/Permissions.php

```diff
@@ -260,4 +260,24 @@ private function grant_additional_caps( array $allcaps ) {
 
 		return $allcaps;
 	}
+
+	/**
+	 * Get all capabilities used in Google Site Kit.
+	 *
+	 * @since 1.x
+	 *
+	 * @return array
+	 */
+	public static function get_capabilities() {
+		return array(
+			self::AUTHENTICATE,
+			self::SETUP,
+			self::VIEW_POSTS_INSIGHTS,
+			self::VIEW_DASHBOARD,
+			self::VIEW_MODULE_DETAILS,
+			self::MANAGE_OPTIONS,
+			self::PUBLISH_POSTS,
+			self::VIEW_POST_INSIGHTS,
```

Comment:
This last one is a meta capability, so whether it would be granted or not depends on which post ID it is called for. By default it just falls back to `VIEW_POSTS_INSIGHTS`, so I think that should be sufficient, since the latter would be possible to be managed through a capability manager plugin.

I think mixing the base capabilities with meta capabilities is rather counter-productive here, so I think this last one should be removed.